test/performance/README.md

```diff
@@ -0,0 +1,82 @@
+# Kueue performance testing
+
+## Measurements
+
+### Job startup latency
+
+How fast do jobs transition from `created` to `started` state?
+Time spent between the transition from `job.CreationTimestamp.Time` to `job.Status.StartTime.Time` state.
+
+### Job startup throughput
+
+Workload admission [rate per second] over minute intervals, Prometheus query:
+
+[To be fine-tuned]
+`max_over_time(sum(rate(kueue_admitted_workloads_total{cluster_queue="{{$clusterQueue}}"}[1m]))[{{$testTimeout}}:5s])`
+
+## How to run the test?
+
+### Prerequisites
+
+1. Deploy [Kueue](https://github.com/kubernetes-sigs/kueue/blob/main/docs/setup/install.md)
+2. Make sure you have `kubectl`, `jq`, [golang version](https://github.com/mikefarah/yq) of `yq` and `go` 
+3. Checkout `Clusterloader2` framework: https://github.com/kubernetes/perf-tests and build `clusterloader` binary:
+
+    * change to `clusterloader2` directory
+    * run `go build -o clusterloader './cmd/'` 
+
+### Test run
+
+* Configure header of `run-test.sh` script:
+
+```
+#!/bin/bash
+
+# Clusterloader home directory (check out https://github.com/kubernetes/perf-tests)
+CL2_HOME_DIR=/Users/johny/perf-tests/clusterloader2
+CL2_BINARY_NAME=clusterloader
+
+# Clusterloader test config directory:
+#
+# kueue        Kueue load test configuration  
+# k8s-job-api  Configuration to test vanilla Kubernetes Job API
+#  
+TEST_CONFIG_DIR=k8s-job-api
+
+# Test iterations
+# number-of-small-jobs number-of-large-jobs job-replica-running-time test-timeout cluster-queue-CPU-quota cluster-queue-memory-quota
+experiments=(          
+    "10 2 0 2s 3m 100 100Gi"
+    "30 6 0 2s 3m 100 100Gi"    
+    "50 10 0 2s 5m 100 100Gi"
+    "100 15 0 2s 10m 100 100Gi" 
+    "150 20 0 2s 15m 100 100Gi"    
+)
+
+# Kubernetes config location, the default context is used
+export KUBECONFIG="$HOME/.kube/config"
+
+# Kubernetes cluster type
+PROVIDER=gke
+
+...
+```
+
+* `chmod +x run-test.sh` and execute the script
+
+### Test results
+
+Every test execution creates a `report_<timestamp>` directory inside `TEST_CONFIG_DIR` with `summary.csv` file, where the following metrics are available:
+
+* P50 Job Create to start latency (ms)
+* P90 Job Create to start latency (ms)
+* P50 Job Start to complete latency (ms)
+* P90 Job Start to complete latency (ms)
+* Max Job Throughput (max jobs/s)
+* Total Jobs
+* Total Pods
+* Duration (s)
```

Comment:
Put the remaining ones after a paragraph that explains that these measure are only shared for reference. Kueue doesn't influence these times.